### ISSUE TITLE
Quick access button

### ISSUE THREAD

**Issue by MangoTornado** [RikkaApps/Shizuku#217]
I was wondering if it would be possible to add a quick access button that starts the wireless adb shizuku method, either combined toggle wireless adb or just starting shizuku.

**Comment by haruue** [comment]
combined toggle of wireless adb is impossible, but I noted that android 12 has a built-in [Wireless debugging] quick toggle in [Quick settings developer tiles].

**Comment by thejaustin** [comment]
> combined toggle of wireless adb is impossible, but I noted that android 12 has a built-in [Wireless debugging] quick toggle in [Quick settings developer tiles]. 
> 
> so maybe user will be able to start Shizuku within only 2 taps.

LADB can automatically turn on wireless debugging from the app. I don't think it's entirely impossible.

**Comment by e-t-l** [comment]
> > combined toggle of wireless adb is impossible
> > [...]
>
> LADB can automatically turn on wireless debugging from the app. I don't think it's entirely impossible.

Correct. `settings set global adb_wifi 1` turns on Wireless Debugging. This can be run from Macrodroid, Tasker, Termux... Shizuku could totally do this.
